### PR TITLE
Add Bag compression option

### PR DESCRIPTION
**Pull request by eborghi10**
Signed-off-by: Emiliano Borghi <emiliano@smartmachine.nz>

Adding Bag compression option.

### CODE REVIEW

File: rosbag_snapshot/src/snapshot.cpp [hunk 1]
```diff
@@ -78,7 +78,9 @@ bool parseOptions(po::variables_map& vm, int argc, char** argv)
      "When in trigger write mode, prepend PREFIX to name of writting bag file")
     ("output-filename,O", po::value<std::string>(), "When in trigger write mode, exact name of written bag file")
     ("topic", po::value<std::vector<std::string> >(),
-     "Topic to buffer. If triggering write, write only these topics instead of all buffered topics.");
+     "Topic to buffer. If triggering write, write only these topics instead of all buffered topics.")
+    ("compression,c", po::value<std::string>(),
```
Comment (dmbuck32) [inline rosbag_snapshot/src/snapshot.cpp]: what about this? 
```suggestion
    ("compression,c", po::value<std::string>()->default_value("uncompressed"),
```

File: rosbag_snapshot/src/snapshot.cpp [hunk 2]
```diff
@@ -120,6 +122,8 @@ bool parseVariablesMap(SnapshotterOptions& opts, po::variables_map const& vm)
   opts.default_duration_limit_ = ros::Duration(vm["duration"].as<double>());
   opts.default_count_limit_ =  vm["count"].as<int32_t>();
   opts.all_topics_ = vm.count("all");
+  if (vm.count("compression"))
```
Comment (dmbuck32) [inline rosbag_snapshot/src/snapshot.cpp]: If we use the default value, then we don't need this check any longer.

**Review by dmbuck32 — CHANGES_REQUESTED** [review]
Let's try using the default value.

File: rosbag_snapshot/src/snapshot.cpp (hunk 2 above)
Comment (eborghi10) [reply rosbag_snapshot/src/snapshot.cpp]: It worked, cool!

**Review by dmbuck32 — APPROVED** [review]
LGTM. 🚀

**Pull request merged**